src/tools/tidy/src/pal.rs

```diff
@@ -78,6 +78,11 @@ const EXCEPTION_PATHS: &'static [&'static str] = &[
     "src/libcore/tests",
     "src/liballoc/tests/lib.rs",
 
+    // The `VaList` implementation must have platform specific code.
+    // The Windows implementation of a `va_list` is always a char
+    // pointer regardless of the target architecture.
+    "src/libcore/va_list.rs",
```

Comment:
Which cfg triggers this exactly? Any of them?

Reply:
`cfg(windows)`. Sadly I don't think we can avoid using it.